src/components/messages/Messages.js

```diff
@@ -171,42 +175,100 @@ const Messages = () => {
     setMessages(filteredArray);
   };
 
+  // useEffect(() => {
```

Comment:
Why was this commented out? Do we know what was it needed for? If not and everything still works fine we should be removing code, not commenting it